### PR TITLE
Move `DisambiguateSameVariableTests` to `VerifyCS`

### PR DESCRIPTION
**Pull request by DoctorKrolic**

### CODE REVIEW

File: src/Analyzers/CSharp/Tests/DisambiguateSameVariable/DisambiguateSameVariableTests.cs [hunk 1]
```diff
@@ -2,79 +2,78 @@
...
 
         [Fact]
         public async Task TestLocalToLocalWithNoMatch()
         {
-            await TestMissingAsync(
+            var code =
 @"class C
 {
-    void M(int a)
+    void M()
```
Comment (CyrusNajmabadi) [inline src/Analyzers/CSharp/Tests/DisambiguateSameVariable/DisambiguateSameVariableTests.cs]: Please don't change test inputs. Rename test, or add new test with the new input.

Comment (DoctorKrolic) [reply src/Analyzers/CSharp/Tests/DisambiguateSameVariable/DisambiguateSameVariableTests.cs]: If you look closely, this is the same test as above, but with different name:
firefox_MgLlkVAtb6
This is 100% unintended, so I just "fixed" it by changing it to what it is meant to do according to its name.

Comment (CyrusNajmabadi) [reply src/Analyzers/CSharp/Tests/DisambiguateSameVariable/DisambiguateSameVariableTests.cs]: Sounds good. In the future, just comment the pr to help the reviewer. Thanks!

**Comment by DoctorKrolic** [comment]
@CyrusNajmabadi Approve + restart failed CI pls

**Review by CyrusNajmabadi — APPROVED** [review]

**Comment by DoctorKrolic** [comment]
@CyrusNajmabadi Restart failed CI pls

**Pull request merged**